### ISSUE TITLE
Groupchat call button becomes green without hang up

### ISSUE THREAD

**Issue by suhr** [qTox/qTox#1434]
Picrelated: https://img.bi/#/86CfUrN!mPA0AoH6oACG5QnJKuPtEp0dsUdycXgdlN75bJUN

**Comment by ProMcTagonist** [comment]
Happens to me too.

**Comment by urras** [comment]
Can also confirm

**Comment by TheLastProject** [comment]
At which point does it happen, exactly? What is the last thing that happens before they change to green when they shouldn't?

**Comment by suhr** [comment]
Can't reproduce anymore.

**Comment by zetok** [comment]
> Can't reproduce anymore.

I could, the last time I've tried audio groupchats, and since code for it hasn't changed, I'm reopening.

I think that @TheSpiritXIII was updating qTox to latest AV API, so perhaps that would be fixed by it when it gets merged.

**Comment by TheSpiritXIII** [comment]
I've been noticing this too, actually.

**Comment by denniszl** [comment]
Hello,

I noticed this bug today on my first use of group calls. I believe this is the issue:

A call has been created already, and someone is added to the group. 
I am not sure about the exact code, however logically onUserListChanged() cannot handle this, as the call button changing has a conditional only on if (peersCount != 1).

Not sure if this code is relevant but in my 30 mins of looking through source code, this is the only place I believe would have cause this, since according to the logs no state has been changed really. Maybe logging should be a little more aggressive too...
